data/announcements/scheduled.yaml

```diff
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-11-10T00:00:00
```

Comment:
this looks like it starts after the event ends...

Reply:
if you want to set a startTime/endTime that is active ~now to let the preview render the banner for content review, that's fine, then can update with the real desired startTime/endTime (I think ending on the last day of the event and starting ~2-3 weeks earlier is typical)

Reply:
(tip: if you want to do that for a preview, also change the PR title to start with “[WIP]” to make it clear that this isn't the final thing).
I'm happy to do a local preview for this and check it looks right for future dates.